chain/network/src/codec.rs

```diff
@@ -26,6 +45,23 @@ impl Encoder<Vec<u8>> for Codec {
         if item.len() > self.max_length as usize {
             Err(Error::new(ErrorKind::InvalidInput, "Input is too long"))
         } else {
+            #[cfg(feature = "performance_stats")]
+            {
+                let stat = get_thread_stats_logger();
+                stat.lock().unwrap().log_add_write_buffer(
+                    item.len() + 4,
+                    buf.len(),
+                    buf.capacity(),
+                );
+            }
+            if buf.capacity() >= MAX_CAPACITY as usize
+                && item.len() + 4 + buf.len() > buf.capacity()
+            {
+                error!("{} throwing away message, because buffer is full item.len(): {} buf.capacity: {}", get_tid(), item.len(), buf.capacity());
```

Comment:
```suggestion
                error!(target: "network", "{} throwing away message, because buffer is full item.len(): {} buf.capacity: {}", get_tid(), item.len(), buf.capacity());
```